test/unit/lib/object_store_test.rb

```diff
@@ -0,0 +1,67 @@
+require "test_helper"
+
+class ObjectStoreTest < ActiveSupport::TestCase
+  setup do
```

Comment:
Given we are making guards against missing fields I think we ought to test the sad path here too, covering happens when this config file is missing keys or contains unexpected keys or values.